src/cljs/chronograph_web/events/routing.cljs

```diff
@@ -1,11 +1,34 @@
 (ns chronograph-web.events.routing
   (:require [re-frame.core :as rf]
-            [chronograph-web.db :as db]))
+            [chronograph-web.http :as http]
+            [chronograph-web.pages.organization.events :as org-events]))
+
+
+(defn- make-get-organization-uri
+  [route]
+  (str "/api/organizations/"
+       (get-in route [:route-params :slug])))
+
+
+(defn- on-navigate
+  [{:keys [handler]
+    :as route}]
+  (case handler
+    :organization-show {:http-xhrio
```

Comment:
The GET request doesn't belong in this namespace. It can be extracted into a function/var in the organization events namespace, and referenced here.

Reply:
Done, moved to the org-events ns.